src/System.ComponentModel.Annotations/src/System/ComponentModel/DataAnnotations/AssociatedMetadataTypeTypeDescriptionProvider.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.ComponentModel.DataAnnotations
+{
+    /// <summary>
+    /// Extends the metadata information for a class by adding attributes and property
+    /// information that is defined in an associated class.
+    /// </summary>
+    public class AssociatedMetadataTypeTypeDescriptionProvider : TypeDescriptionProvider
```

Comment:
@ajcvickers I addressed the [code review](https://github.com/dotnet/corefx/pull/33312#pullrequestreview-174629736)